### PR TITLE
add podman container and image command

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: docs/podman-container.1.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+% podman-container "1"
+
+## NAME
+podman\-container - Manage containers
+
+## SYNOPSIS
+**podman container SUBCOMMAND [OPTIONS]
```
Comment (TomSweeneyRedHat) [inline docs/podman-container.1.md]: Don't they need to provide a CONTAINER too?

**Comment by rhatdan** [comment]
podman container attach == podman attach.  It is just a different interface into the same calls.  I want to add a podman container cleanup though which we could call from conmon when we want to cleanup containers running in daemon mode.

podman image COMMAND is also the same as podman COMMAND.

File: docs/podman-container.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/podman-container.1.md]: That depends on the subcommand

podman container ls

Does not require a container.

File: docs/podman-container.1.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+| rm       | [podman-rm(1)](podman-rm.1.md)            | Remove one or more containers.                                                 |
+| run      | [podman-run(1)](podman-run.1.md)          | Run a command in a container.                                                  |
+| start    | [podman-start(1)](podman-start.1.md)      | Starts one or more containers.                                                 |
+| stats    | [podman-stats(1)](podman-stats.1.md)      | Display a live stream of one or more container's resource usage statistics.    |
+| stop     | [podman-stop(1)](podman-stop.1.md)        | Stop one or more running containers.                                           |
+| top      | [podman-top(1)](podman-top.1.md)          | Display the running processes of a container.                                  |
+| umount   | [podman-umount(1)](podman-umount.1.md)    | Unmount a working container's root filesystem.                                 |
+| unmount  | [podman-umount(1)](podman-umount.1.md)    | Unmount a working container's root filesystem.                                 |
```
Comment (TomSweeneyRedHat) [inline docs/podman-container.1.md]: link should go to unmount not umount for this one.

File: docs/podman-container.1.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/podman-container.1.md]: Gotcha, so it's buried away as part of the options.  Wish we could tease it out somehow on this page,  but I'm not thinking of a nice way to do that without making a huge mess.  Probably best off leaving it to the manpage links you've provided.

**Comment by TomSweeneyRedHat** [comment]
Should we add these commands to commands.md?

File: docs/podman-container.1.md (hunk 2 above)
Comment (rhatdan) [reply docs/podman-container.1.md]: There is no poduman unmount man page, only umount
unmount is an alias for umount.

**Comment by rhatdan** [comment]
Added entries to commands.md

File: docs/podman-container.1.md (hunk 2 above)
Comment (TomSweeneyRedHat) [reply docs/podman-container.1.md]: Ugh, think I even asked this in a prior review, back to my hole.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**